### ISSUE TITLE
[BUG]paint only in mask area function missing

### ISSUE THREAD

**Issue by liangwei191** [hlky/stable-diffusion-webui#296]
The function in former version which can modifity only in a masked area seems missing.
It is removed? It is a very important function for fixing detail.Hope could put it back.
Thanks.

**Comment by nderscore** [comment]
Fix is in #294

You can get `ui_functions.py` from [here](https://github.com/hlky/stable-diffusion-webui/blob/3839721ebc5ee42d12ae1aa41e2a7e1a6956095d/frontend/ui_functions.py) if you want it early

**Comment by hlky** [comment]
Thanks @nderscore #294 was merged to master of dev repo.

@liangwei191 you'll need to wait for the changes to be synced to main repo unless you understand how to update from dev repo -> main repo yourself like @nderscore is saying 

```
Dev repo is set up to be easy for developers so the structure is different.
Changes are synced to main repo.
GitHub Actions puts things in the right places.
If you can't tell where things go check .github/sync.yml
```
